routes/api/v1/playlists.js

```diff
@@ -26,7 +27,23 @@ router.post('/', (request, response) => {
     })
   } else
   response.status(422).json({error: 'Playlist not created, please enter a title.'})
-
 });
 
+router.delete('/:id', (request, response) => {
+  let id = request.params.id
+  database('playlists').where('id', id).del()
+  .then(playlist => {
+    if (playlist > 0) {
+      response.status(204).send();
+    } else {
+      response.status(404).json({
+        error: `Could not find playlist with id ${id}`
+      });
+    }
+  })
+  .catch(error => {
```

Comment:
We just need to update the error message that is being sent back here, but we can make a separate card for that so it is consistent between both favorites and playlists